src/components/molecules/Cv/components/DateDescription.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from 'react';
+import { format } from 'date-fns';
+import { Text, BoxProps } from '../../../quarks';
+
+export const DateDescription: FC<ICvCreate> = ({ createdAt, ...props }) => (
+  <Text as="p" mb={3} fontScale={2} color="2" {...props}>
+    Aplikacja wysłana: {format(new Date(createdAt!), 'dd.MM.yyyy')}, godz.{' '}
```

Comment:
Co to za sekretny wykrzyknik w `createdAt`? :)

Reply:
nie wiem, ktoś wziął i wpisał :O